src/components/Authentication/Login.js

```diff
@@ -72,6 +77,7 @@ function Login(props) {
   const classes = useStyles();
   Auth.currentAuthenticatedUser()
     .then(user => {
+      console.log(user);
```

Comment:
🔪

Reply:
@superzzp can we clean up any remaining console.logs

Reply:
updated